recipe/meta.yaml

```diff
@@ -1,44 +1,41 @@
 {% set version = "4.0.1" %}
-{% set sha256 = "cce7b6d20522849301727f81282201d609553103ac0b09162cf28d102efb9709" %}
 {% set major = version.rpartition('.')[0] %}
 
 package:
+  # must not match any outputs for requirements to be handled correctly
   name: openmpi-mpi
   version: {{ version }}
 
 source:
   fn: openmpi-{{ version }}.tar.bz2
   url: https://www.open-mpi.org/software/ompi/v{{ major }}/downloads/openmpi-{{ version }}.tar.bz2
-  sha256: {{ sha256 }}
+  sha256: cce7b6d20522849301727f81282201d609553103ac0b09162cf28d102efb9709
 
 build:
-  number: 1
+  number: 2
   skip: True  # [win]
 
 outputs:
   - name: openmpi
-    script: build-openmpi.sh
+    script: build-mpi.sh
     build:
       run_exports:
         - {{ pin_subpackage('openmpi', min_pin='x.x.x', max_pin='x.x') }}
-
     requirements:
       build:
-        - perl 5.26.2
-      host:
-        - {{ compiler('fortran') }}
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+        - {{ compiler('fortran') }}
         - make  # [unix]
+        - perl 5.26.2
```

Comment:
perl is not a host dependency. This should go back to build

Reply:
Of course! Look again to the diff, note the line below this one: `hosts = []`.

Reply:
For history, compilers were placed in host to fix references to compiler paths in 630319f2db912205a84613c60ecc6d1727360331 . I believe this is made obsolete by 1015075f4aa5995e836921db2aa62f249911ba8b so this change should be okay.